examples/diff/index.html

```diff
@@ -89,28 +102,32 @@
      */
     function renderDiff() {
 
+      // Do not render if there is no published value.
+      if (!publishedVersion) {
+        diffTextEl.innerHTML = '<i>Entry not yet published.</i>';
+        return;
+      }
+
       // Do not render if there is no diff.
-      if (publishedValue == draftTextEl.value) {
+      if (publishedValue === draftTextEl.value) {
         diffTextEl.innerHTML = '<i>Draft and published value are equal.</i>';
-        //diffTextEl.parentNode.style.display = 'none';
         return;
       }
 
       // Clear diff DOM element.
       diffTextEl.innerHTML = '';
 
-      //diffTextEl.parentNode.style.display = 'block';
-
       var diff = JsDiff.diffWordsWithSpace(publishedValue, draftTextEl.value);
 
+      var className;
```

Comment:
I would move `var className` and `var spanEl` inside the function passed to `forEach` since they are only used in there.